src/mlpack/methods/ann/augmented/tasks/add.hpp

```diff
@@ -0,0 +1,82 @@
+/**
+ * @file add.hpp
+ * @author Konstantin Sidorov
+ *
+ * Definition of the AddTask class, which implements a generator of
+ * instances of binary addition task.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_AUGMENTED_TASKS_ADD_HPP
+#define MLPACK_METHODS_AUGMENTED_TASKS_ADD_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /* Artificial Neural Network */ {
+namespace augmented /* Augmented neural network */ {
+namespace tasks /* Task utilities for augmented */ {
+class AddTask
```

Comment:
Can you put a new line between the last namespace and the class. Also It would be great if you could add some comments? I guess adding an example with that also shows the output might be helpful, something like: https://github.com/mlpack/mlpack/blob/master/src/mlpack/core/optimizers/gradient_descent/gradient_descent.hpp. What do you think?